### DISCUSSION TITLE
Will it work without analog needles setup?

### DISCUSSION THREAD

**Discussion by Trilogy2021** [jomjol/AI-on-the-edge-device#1490] [Q&A] [answered]
Hi all,

i wanted to set up my esp32 and was struggling once i came to the step to set up the analog needles. The problem is that the only analog needle on my meter is covered by a (from my comunity installed) remote reader. Is it possible to skip this step?

-Christian
![IMG_1692](https://user-images.githubusercontent.com/98818204/205626629-5ba18e87-d76c-453a-8731-650c5ba60111.jpg)

**Comment by jomjol** [comment, marked as answer]
Here you can dis/enable the analog counters:
![grafik](https://user-images.githubusercontent.com/30766535/205707684-3258ee5d-89ff-47e2-979b-bbe50be954fd.png)

**Reply by Trilogy2021** [reply]
Sorry for the late response but thank you. I missed this check box.
